hack/tool/backup_restore/cmd/restore.go

```diff
@@ -115,16 +115,15 @@ func restore(ctx context.Context) error {
 func presetTFBackendNS() {
 	backendNS := os.Getenv(app.TFBackendNS)
 	if backendNS != "" {
-		goto end
+		log.Printf("use the `TERRAFORM_BACKEND_NAMESPACE` environment variable: %s", backendNS)
```

Comment:
If not return, `backendNS` will be overwriten.

Reply:
Oh, my bad.